pyiron/lammps/control.py

```diff
@@ -147,49 +147,66 @@ def calc_md(self, temperature=None, pressure=None, n_ionic_steps=1000, time_step
             delta_temp (float): Thermostat timescale, but in your Lammps time units, whatever those are. (DEPRECATED.)
             delta_press (float): Barostat timescale, but in your Lammps time units, whatever those are. (DEPRECATED.)
         """
-        if delta_temp is not None:
-            warnings.warn("WARNING: `delta_temp` is deprecated, please use `temperature_damping_timescale`")
-            temperature_damping_timescale = delta_temp
-        if delta_press is not None:
-            warnings.warn("WARNING: `delta_press` is deprecated, please use `pressure_damping_timescale`")
-            pressure_damping_timescale = delta_press
-
-        time_rescaling_factors = {'metal': 1e-3,
-                                  'si': 1e-12,
-                                  'cgs': 1e-12,
-                                  'real': 1,
-                                  'electron': 1}
-
+        # Conversion factors for transfroming pyiron units to Lammps units
+        fs_to_ps = 1.e-3
+        fs_to_s = 1.e-12
+        GPa_to_bar = 1.e4
+        GPa_to_Pa = 1.e9
+        GPa_to_barye = 1.e10
+        GPa_to_atm = 9869.232667160128  # digits c.f. http://www.kylesconverter.com/pressure/gigapascals-to-atmospheres
```

Comment:
Can not we do the unit conversion with scipy constants?

Reply:
Certainly. I see to do it like this:

`import scipy.constants as sconst

fs_to_ps = sconst.femto / sconst.pico
fs_to_s = sconst.femto / 1.
GPa_to_bar = sconst.giga * 1. / sconst.bar
GPa_to_Pa = sconst.giga 
GPa_to_barye = sconst.giga * 1. / (1.e-6 * sconst.bar)  # Lammps is in "barye"
GPa_to_atm = sconst.giga * 1. / sconst.atm`

...but to me this is quite ugly looking compared to just putting the number. Is there a more elegant way to phrase it?

Reply:
I am not aware if there is a more elegant way - so I guess for now we can just implement it like this.

Reply:
Amended.